lib/quicly.c

```diff
@@ -5586,12 +5583,11 @@ int quicly_receive(quicly_conn_t *conn, struct sockaddr *dest_addr, struct socka
     if ((ret = decrypt_packet(header_protection, aead.cb, aead.ctx, &(*space)->next_expected_packet_number, packet, &pn,
                               &payload)) != 0) {
         ++conn->super.stats.num_packets.decryption_failed;
-        QUICLY_PROBE(CRYPTO_DECRYPT, conn, conn->stash.now, pn, NULL, 0);
+        QUICLY_PROBE(PACKET_DROPPED, conn, conn->stash.now, "payload_decrypt_error", pn);
```

Comment:
I think we might want to have a dedicated probe for decryption failure rather than using a string that identifies a cause. This is an important event that we also count, it's nice to have a way of just probing those.

Something like: `QUICLY_PROBE(PACKET_DECRYPTION_FAILURE, conn, conn->stash.now, pn)`.

Reply:
Sounds good to me. Addressed in: 3d30865